modules/heat_conduction/test/tests/verify_against_analytical/tests

```diff
@@ -17,23 +17,49 @@
     design = "jacobian_definition.md"
     issues = "#5658 #12633"
   [../]
+  [./ad_1D_transient_jacobian]
+    type = 'PetscJacobianTester'
+    input = 'ad_1D_transient.i'
+    ratio_tol = 1e-7
+    difference_tol = 1e-5
+    run_sim = True
+    petsc_version = '>=3.9'
+    method = 'OPT'
+    cli_args = 'Executioner/num_steps=5 Outputs/exodus=false'
+    requirement = 'AD heat conduction and the Jacobian shall be beautiful'
+    design = "jacobian_definition.md"
+    issues = "#5658 #12633"
+  [../]
 
   [./2D_steady_state]
     type = 'Exodiff'
-    input = '2d_steady_state_final_prob.i'
-    exodiff = '2d_steady_state_final_prob_out.e'
+    input = '2d_steady_state.i'
+    exodiff = '2d_steady_state_out.e'
     abs_zero = 1e-8
     requirement = "Heat conduction shall match the answer from an analytical solution"
     design = "HeatConduction.md"
     issues = "#8194"
   [../]
-  [./ad_2D_steady_state_ad_mat]
+  [./ad_2D_steady_state]
     type = 'Exodiff'
-    input = 'ad_2d_steady_state_final_prob.i'
-    exodiff = 'ad_2d_steady_state_final_prob_out.e'
+    input = 'ad_2d_steady_state.i'
+    exodiff = 'ad_2d_steady_state_out.e'
     abs_zero = 1e-8
     requirement = "Heat conduction from an AD kernel shall get the same answer as a traditional kernel"
     design = "jacobian_definition.md"
     issues = "#5658 #12633"
   [../]
+  [./ad_2D_steady_state_jacobian]
+    type = 'PetscJacobianTester'
+    input = 'ad_2d_steady_state.i'
+    ratio_tol = 1e-7
+    difference_tol = 1e-5
+    run_sim = True
+    petsc_version = '>=3.9'
+    method = 'OPT'
```

Comment:
What's the reason for the `method` specification?

Reply:
It takes too long in dbg